pkg/master/master.go

```diff
@@ -255,6 +260,46 @@ func (m *Master) InstallAPIs(c *Config) {
 		}
 		allGroups = append(allGroups, group)
 	}
+
+	// Install autoscaling unless disabled. Requires 'horizontalpodautoscalers' resource in extensions group to be enabled.
+	if !m.ApiGroupVersionOverrides["autoscaling/v1"].Disable && autoscalerStorage != nil && autoscalerStatusStorage != nil {
```

Comment:
I think it will be better to explicitly fail if autoscaling/v1 is enabled but autoScalerStorage is nil.